.eslintrc.json

```diff
@@ -56,6 +58,18 @@
     ],
     "quotes": ["error", "single", { "allowTemplateLiterals": true, "avoidEscape": true }]
   },
+  "settings": {
+    "import/core-modules": 
+      [
+        "googlesitekit-data",
+        "googlesitekit-modules",
+        "googlesitekit-api",
+        "googlesitekit-widgets",
+        "@wordpress/i18n__non-shim",
+        "@wordpress/api-fetch__non-shim",
+        "tests/js/utils"
```

Comment:
This is a quick workaround to not get lots of errors. Next, I'll look at resolvers

This workaround could work well as these are only used in tests *and* the tests simply won't run if not correctly imported *but* it's nice to have this plugin working to speed up development time when writing tests :smile:

Reply:
Is this still an issue or did you get the resolvers working? 🙂